apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.jsx

```diff
@@ -24,13 +25,16 @@ import Receive2NoDevice from "~/renderer/components/Receive2NoDevice";
 import { renderVerifyUnwrapped } from "~/renderer/components/DeviceAction/rendering";
 import type { StepProps } from "../Body";
 import type { AccountLike } from "@ledgerhq/types-live";
+import { track } from "~/renderer/analytics/segment";
 import Modal from "~/renderer/components/Modal";
 import Alert from "~/renderer/components/Alert";
 import ModalBody from "~/renderer/components/Modal/ModalBody";
 import QRCode from "~/renderer/components/QRCode";
 import { getEnv } from "@ledgerhq/live-common/env";
 import AccountTagDerivationMode from "~/renderer/components/AccountTagDerivationMode";
 import byFamily from "~/renderer/generated/StepReceiveFunds";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { LOCAL_STORAGE_KEY_PREFIX } from "./StepReceiveStakingFlow";
```

Comment:
@sarneijim @LFBarreto Any reason for using local storage instead of the app.json?

Reply:
I think app json is only for store "secure" things. But it needs to be confirmed by @LFBarreto when he goes back.

Reply:
for all "popup" dismiss states of the app we use localstorage for it, we do that for terms and conditions and all the other dismissable popup modals